python/cudf/cudf/core/column/string.py

```diff
@@ -303,11 +299,9 @@ def len(self, **kwargs):
         dtype: int32
         """
 
-        return self._return_or_inplace(
-            cpp_count_characters(self._column), **kwargs,
-        )
+        return self._return_or_inplace(cpp_count_characters(self._column),)
```

Comment:
Did black add the trailing comma here? It looks a bit funny

Reply:
I think it's more that black was OK with it. pre-commit has really spoiled me to where I don't bother writing pretty code anymore :D 

Fixed